packages/core/useVModel/index.ts

```diff
@@ -54,12 +61,16 @@ export function useVModel<P extends object, K extends keyof P, Name extends stri
   event = eventName || event || `update:${key}`
 
   if (passive) {
-    const proxy = ref<P[K]>(props[key!])
+    const defaultProxy = deep ? JSON.parse(JSON.stringify(props[key!])) : props[key!]
```

Comment:
I am a worry about the clone here. As user might pass some object/instance that are not serializable (or not intended to be cloned) and introduce some performance cost. I don't think it's a proper solution here.

Reply:
@antfu Thanks. i clone it is because  the `props`  and `proxy` data share the same mem address:
```ts
watch(proxy, (v) => {
  if (v !== props[key!]) // always equal
    _emit(event, v)
}, {
  deep,
})
```
maybe we can skip it  when user  setup the deep option ?  Or do you have some other proper solution ? 😊

```ts
watch(proxy, (v) => {
  if (v !== props[key!] || deep) // skip when user  setup the deep options
    _emit(event, v)
}, {
  deep,
})
```

Reply:
Actually I am wondering what's the use case here? If you changed nested props, as they are the same reference for both child and parent, it should be no problem of passing the reactivity.

Reply:
@antfu this purpose of  :
```ts
watch(proxy, (v) => {
  if (v !== props[key!]) // changed nested props,always equal. so can't trigger emit
    _emit(event, v)
}, {
  deep,
})
```

Reply:
If you only want it to trigger the emit everytime, maybe this could be a solution

```ts
watch(proxy, (v) => {
  if (v !== props[key!] || deep) // skip when user  setup the deep options
    _emit(event, v)
}, {
  deep,
})
```

but I am just not really understand why you need this

Reply:
@antfu  just want to fix #613 😂

Reply:
ok, let's go with the approach above then. Thanks

Reply:
@antfu Thanks~